src/odemis/driver/semcomedi.py

```diff
@@ -1705,11 +1724,16 @@ def _acquisition_run(self):
                         nfailures += 1
                         if nfailures == 5:
                             logging.exception("Acquisition failed %d times in a row, giving up", nfailures)
+                            self._scanner.state._set_value(
+                                model.HwError("DAQ board non responsive, restart the acquisition to try a new recovery."),
```

Comment:
I don't fully get this sentence. Should it be:
"DAQ board non responsive, try restarting the acquisition."?

Reply:
Well, restarting the acquisition is not necessarily going to fix the issue. It'll just force another attempt to use the DAQ board (and from the current experience, most likely fail again). Maybe I should be more honest/pessimistic. What do you think of:
"DAQ board non responsive, try restarting the acquisition or power-cycling the computer."

Reply:
Sounds good maybe this is better (just slightly less pessimistic):
"DAQ board currently non responsive, try restarting the acquisition or power-cycling the computer."